adapters/primitive-adapters/build.gradle

```diff
@@ -0,0 +1,12 @@
+apply plugin: 'org.jetbrains.kotlin.jvm'
+
+archivesBaseName = 'sqldelight-primitive-adapters'
```

Comment:
I'm not sure how I feel about the name "primitive" tbh, as Kotlin doesn't have primitive types.

Reply:
Perhaps we should name it `sqldelight-sqlite-adapters` per https://github.com/cashapp/sqldelight/pull/2121/files#r546376383?

Reply:
yes, given our discussion yesterday the idea is that dialects will have their own repo, and part of it would be a set of adapters - and in this case these are the SQLite adapters, where something like postgres will have none of these since you should be using the correct column type :+1: